microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md

```diff
@@ -24,43 +24,32 @@ To better meet the needs of your users with [color vision deficiency](https://ww
 <!-- ====================================================================== -->
 ## Open the Rendering tool
 
-To simulate a vision deficiency being applied for your web product, open the [Rendering Tools](../rendering-tools/index.md).
+To simulate how a person with a vision deficiency would see your web page, open the [Rendering Tools](../rendering-tools/index.md).
 
-1.  To open the Rendering tool, select the `...` menu item in the toolbar.
-1.  Select **More tools**.
-1.  Select **Rendering**.
+1.  Click **Customize and control DevTools** (`...`) in the toolbar.
+1.  Go to **More tools** > **Rendering** to open the **Rendering** panel in the drawer.
 
-    :::image type="content" source="../media/getting-to-the-rendering-tools.msft.png" alt-text="Opening the Rendering tool." lightbox="../media/getting-to-the-rendering-tools.msft.png":::
+    > [!div class="mx-imgBorder"]
```

Comment:
In Preview in VS Code, removing > caused the preview to be truncated here.  So, > seems required (probably).  I haven't tested that in the built/rendered docs, only in VS Code locally.

You could remove this tagging, or apply it to other images in this PR.